BZ2021867: Extend the IMPORTANT warning (#38915)
openshift/openshift-docs: modules/technology-preview.adoc

might not be functionally complete. Red Hat does not recommend using them
in production. These features provide early access to upcoming product
features, enabling customers to test functionality and provide feedback during

> the development process. The GA is available through Virtualization. You must not install both the Kubernetes NMState Operators at the same time.

Reviewer: Should this be Red Hat Virtualization?

Author: acked

> the development process. The GA is available through {rh-virtualization-first}. Do not install both the Kubernetes NMState Operators at the same time.

Reviewer 2: I'm not sure what you mean:
* "The GA": My best guess here is that this Operator is GA in RH Virtualization. Is that what you're referring to?
* "both the Kubernetes NMState Operators": Do you mean the TP version (in OCP) and the GA version ( in RH Virtualization)?

Author: @ctauchen The answers for both the questions is `yes`

- Yes. I am referring to `Operator is GA in RH Virtualization.`
- Yes. I mean `the TP version (in OCP) and the GA version ( in RH Virtualization)`
 I think it might be kind of confusing right now. Should I explicitly mention it in that way-  both the operators - one in OCP and one in RHV?

Reviewer 2: Ok. In that case, perhaps something like this:

`A fully supported version of {FeatureName} is available with {rh-virtualization-first}.`

I realize now that the second sentence could be clearer. Right now reads two ways: 1) You must not perform the installation of both Operators at the same time (that is, you must install them one after the other) or 2) You must not have both Operators installed simultaneously.

I suspect it's 2) we're going for, but that's not clear from the BZ. If it is in fact 2), then perhaps something like:

`You must not use {FeatureName} in both {product-name} and {rh-virtualization-first} at the same time.'

Or, if more accurate, perhaps say that this is not supported.

Author: Done.

> the development process. The GA is available through Red Hat Virtualization. You must not install both the Kubernetes NMState Operators at the same time.

Reviewer 3: ```suggestion
the development process. The GA is available through {rh-virtualization-first}. Do not install both the Kubernetes NMState Operators at the same time.
```

Red Ht Virtualization has an attribute in the common-attributes.adoc file for us to use. Just clarified the second sentence for easier reading. :)

Author: Done
